docs/styles/docs.scss

```diff
@@ -1,66 +1,116 @@
-.docs__container {
+@use 'tools' as *;
+
+.docs {
   --nav-width: 200px;
   --site-gap: 40px;
-  display: grid;
-  grid-template-columns: 1fr;
-  max-width: calc(var(--content-max-width) + var(--nav-width));
-  width: 100%;
-  align-items: flex-start;
-  margin: auto;
-  padding: 0 20px;
-  gap: var(--site-gap);
+  --site-max-width: calc(var(--content-max-width) + var(--nav-width));
+
+  &__container {
+    display: grid;
+    grid-template-columns: 1fr;
+    max-width: calc(var(--content-max-width) + var(--nav-width));
+    width: 100%;
+    align-items: flex-start;
+    margin: auto;
+    padding: 0 20px;
+    gap: var(--site-gap);
+  
+    @media (min-width: breakpoint(tablet)) {
+      grid-template-columns: [side-start] var(--nav-width) [side-end content-start] 1fr [content-end];
+    }
+    
+    &__content {
+      overflow: auto;
+      max-width: 100%;
+      margin: 0;
+      padding: 0;
+  
+      > h1 {
+        margin-block-start: var(--site-gap);
+      }
+  
+      @media (min-width: breakpoint(tablet)) {
+        grid-column: content-start / content-end;
+      }
+  
+      @include gradient-links;
+    }
+  }
+
+  &__primary-nav {
+    --max-width: var(--site-max-width);
+    height: 4rem;
+    display: grid;
+    align-items: center;
+
+    &__content {
+      width: 100%;
+      align-items: center;
+      margin: auto;
+      padding: 0 1rem;
+    }
+
+    a:not(.docs__home-link) {
+      @include gradient-link;
+      color: white;
+      text-decoration: none;
+    }
+  }
 
-  @media (min-width: $tablet) {
-    grid-template-columns: [side-start] var(--nav-width) [side-end content-start] 1fr [content-end];
+  &__home-link, &__home-link:visited {
+    color: white;
+    text-decoration: none;
+    font-weight: bold;
+    font-size: 32px;
+    margin-right: auto;
   }
 
-  .docs__nav {
+
+  &__side-nav {
     list-style: none;
```

Comment:
wasn't changed, but let's make sure that the `<ul>`/`<ol>` that is selected here has `role="list"` on it. In some screenreaders, `list-style: none;` removes the implicit role.